### ISSUE TITLE
pairs() doesn't allow visualizing sampler diagnostics

### ISSUE THREAD

**Issue by Raoul-Kima** [stan-dev/rstan#719]
Pairs(stanfit) doesn't seem to allow me to add sampler diagnostics such as "energy__" to the plotted parameters (via argument "pars"). This would be very useful.

Rstan version: 2.19.2
R version 3.6.1
Windows 7

**Comment by petrelharp** [comment]
I agree - and for more context: I recently was trying to do this same thing, since in [the documentation on warnings](https://mc-stan.org/misc/warnings.html), it says that in the case of low BFMI,
> Look at the pairs plot to see which primitive parameters are correlated with the energy__ margin.

... but how to do this? You can do `pairs(fit, pars=(...), condition="energy__")`, which will plot in the upper/lower triangle the top/bottom half of transitions partitioned by energy, but (in my case?) it's pretty hard to tell what's correlated with energy from that.

If this isn't fixed, some more info in the documentation would be nice.
